packages/core/types/common.d.ts

```diff
@@ -8,49 +8,23 @@ export interface IConfig {
   autoNotify?: boolean;
   appVersion?: string;
   appType?: string;
-  endpoint?: string;
-  sessionEndpoint?: string;
   endpoints?: { notify: string, sessions: string };
   autoCaptureSessions?: boolean;
   notifyReleaseStages?: string[];
   releaseStage?: string;
-  maxEvents?: number;
   maxBreadcrumbs?: number;
-  consoleBreadcrumbsEnabled?: boolean;
-  networkBreadcrumbsEnabled?: boolean;
-  navigationBreadcrumbsEnabled?: boolean;
-  interactionBreadcrumbsEnabled?: boolean;
```

Comment:
I'm assuming that these options are no longer used/are specified elsewhere

Reply:
yeah these are all browser-specific so they're added in [here](https://github.com/bugsnag/bugsnag-js/pull/382/files#diff-3ef064881d5d7b974f81782769d32e82R3)